GargasMAOverview.md

```diff
@@ -0,0 +1,41 @@
+﻿# The Acoustics of Smiling Voice
+
+## **Abstract**
+
+## **Introduction**
+
+General information about the aim of the thesis, introduction to the subject of the thesis, information about its structure and what would be presented in the following chapters.
+
+## **Chapter 1. Theory**
+
+1.Phonetics
```

Comment:
In order for the list to be rendered correctly, you have to put a space after the period. Look at the output: https://github.com/nlp-ug-2018-19/MA-Thesis-Overview/blob/gargasaleksandra-patch-2/GargasMAOverview.md Is this what you expected? Please think about sub- and subsubsections - make sure that they are properly indented in your plan.